src/components/collections/form/field.jsx

```diff
@@ -5,8 +5,8 @@ import classNames from 'classnames';
 
 function getClasses(props) {
     let classes = {
-        disabled: props.state === 'disabled',
-        error: props.state === 'error',
```

Comment:
What is the point to have 'disabled' and 'error' rather than single state="disabled" ?